### PR TITLE
library website searches based on screen scraping

### LINKED ISSUE #3: Write a search class to fetch library website results
The library website search is backed by a solr instance in our solr cloud.

Not sure if there is any API made available to get search results from Drupal, may be worth investigating.

### PR DESCRIPTION
**Pull request by drh-stanford**
This PR fixes #3. It's based on screen scraping so it's fragile. An API-based solution is being investigated...

- [x] handle lacking number of hits
- [x] handle no results case (broken out into https://github.com/sul-dlss/sul-bento-app/issues/59)

### After

screen shot 2017-09-13 at 3 17 58 pm

### CODE REVIEW

File: app/services/library_website_search_service.rb [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+    options[:response_class] ||= Response
+    super
+  end
+
+  class Response < AbstractSearchService::Response
+    HIGHLIGHTED_FACET_FIELD = 'document_type_facet'.freeze
+    QUERY_URL = Settings.LIBRARY_WEBSITE_QUERY_API_URL.freeze
+
+    def total
+      999_999_999 # TODO: library web scrape doesn't provide the total number of hits
```
Comment (jkeck) [inline app/services/library_website_search_service.rb]: @jvine what would you like us to do here?  I feel like we'll never get the total # of results w/ the screen scraping method.  Should this just not show any number?

Comment (drh-stanford) [reply app/services/library_website_search_service.rb]: I've added afd5ee8 which does not show a number (e.g., "See all Library website results")

Comment (jvine) [reply app/services/library_website_search_service.rb]: @drh-stanford that's fine for now - frankly at the moment the Drupal bento can't get the total number of Drupal website results either.

**Comment by camillevilla** [comment]
This looks good. But we'll need to implement some behavior for library website searches that don't return results. Maybe CSS `display:none` the `library-website` div? 

## No results
no results

**Comment by jvine** [comment]
@camillevilla I think an explicit block with a title and "no results" is the best approach. Should this be a separate ticket?

**Comment by drh-stanford** [comment]
This is what's happening now for articles+ in bento...
screen shot 2017-09-14 at 3 36 21 pm

**Comment by jvine** [comment]
Okay cool. Can we simplify, though? 

screen shot 2017-09-14 at 4 01 37 pm

**Comment by drh-stanford** [comment]
Ok, I've broken out the No results handling into its own ticket (https://github.com/sul-dlss/sul-bento-app/issues/59) as we'll need to do that for each bento search.

**Pull request merged**